tests/ra_aid/test_llm.py

```
@@ -114,6 +125,16 @@ def test_initialize_openai(clean_env, mock_openai):
         model="gpt-4"
     )
 
+def test_initialize_gemini(clean_env, mock_gemini):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_initialize_gemini`

Add return type annotation: `None`

(ANN201)